test/Microsoft.Azure.Devices.Edge.Test.TempSensor/Microsoft.Azure.Devices.Edge.Test.TempSensor.csproj

```diff
@@ -36,6 +36,9 @@
 
   <ItemGroup>
     <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="2.3.4" />
+    <PackageReference Include="Serilog" Version="2.8.0" />
```

Comment:
Serilog needs to be added to `THIRDPARTYNOTICES`. (Seems the `load-gen` example has already been using it without adding it to the file.)

Reply:
Referencing the Serilog package isn't necessary; the project builds without it so I'll remove it. However I'm also referencing `Serilog.Sinks.File` which isn't in THIRDPARTYNOTICES, so I'll add it.